frontend/scss/pages/_home_page.scss

```diff
@@ -199,6 +199,48 @@
   margin-top: 50px;
 }
 
+.products-component-container {
+  margin: 0 auto;
+}
+
+.products-component {
```

Comment:
i generally try to avoid naming things with a name that implies implementation, so i'd name this `product-summary`. Same thing with JS, i wouldn't call something like 'somethingArray', I'd call it 'somethings' or some other name